### PR TITLE
Add check to see if repo is up-to-date with origin

### PR DESCRIPTION
**Pull request by AlanGabbianelli**
This is implemented by checking the `git diff` between the local master
branch and the remote master branch.

### CODE REVIEW

File: lib/doctor/checkup.rb [hunk 1]
```diff
@@ -14,10 +14,15 @@ def call
     end
 
     def checkup
+      up_to_date? if checkups.include?(:up_to_date)
       installed? if checkups.include?(:installed)
       running? if checkups.include?(:running)
     end
 
+    def up_to_date?
+      @up_to_date ||= !(system "git -C ~/govuk/govuk-docker diff master origin/master | cat 1>/dev/null")
```
Comment (thomasleese) [inline lib/doctor/checkup.rb]:
```suggestion
      @up_to_date ||= !(system "git -C #{GovukDocker::Paths.govuk_docker_dir} diff master origin/master | cat 1>/dev/null")
```

File: lib/doctor/doctor.rb [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
       docker_compose: docker_compose_messages
     }
   end
 
+  def self.govuk_docker_messages
+    {
+      up_to_date: "✅ govuk-docker is up-to-date",
+      outdated: <<~HEREDOC,
+        ❌ govuk-docker is outdated.
+        You should pull the lates version from https://github.com/alphagov/govuk-docker/.
```
Comment (thomasleese) [inline lib/doctor/doctor.rb]:
```suggestion
        You should pull the latest version from https://github.com/alphagov/govuk-docker/.
```

File: lib/govuk_docker_cli.rb [hunk 3]
```diff
@@ -56,7 +56,14 @@ def compose(*args)
 
   desc "doctor", "Various tests to help diagnose issues when running `govuk-docker`"
   def doctor
-    puts "Checking dnsmasq"
+
+    puts "Chechink govuk-docker"
```
Comment (thomasleese) [inline lib/govuk_docker_cli.rb]:
```suggestion
    puts "Checking govuk-docker"
```

Comment (AlanGabbianelli) [reply lib/govuk_docker_cli.rb]: 🤦‍♂

**Pull request merged**